validator.py

```diff
@@ -88,27 +88,30 @@
     return facts
 
 
-class MQStatus(object):
-    """Class used to track the status of the producer/consumer clients."""
-    mqc_connected = False
-    mqp_connected = False
+mq_conn_status = {"consumer": False, "producer": False}
+
+
+async def ensure_connected(direction, level=0):
+    if level > 5:
```

Comment:
Do we really want to stop connecting after 5 attempts? If it's running as part of the loop, it might not force the pod to fall over and then we'd just stop consuming/producing.

Reply:
Are you thinking we should keep trying indefinitely?

Reply:
I would think so. The validator should always be listening to messages on the queue. I can't think of a time we would want it to quit and not process incoming messages. 
We could have it just blow up the whole pod and spin up again, but that feels messy. Outside of maybe giving us some sort of alert that it keeps recreating.